Pipfile

```diff
@@ -37,9 +38,9 @@ scipy = "==1.6.2"
 seaborn = "*"
 sqlalchemy = "*"
 statsmodels = "*"
-tensorboard = "<2.0"
-torch  = "*"
-torchvision= "*"
+tensorboard = "==1.15.0"
+torch  = "==1.8.1"
```

Comment:
can we pin the torch version to 1.9.0 to preserve the version currently listed in the imagestream annotations? (is 1.8.1 essential for some reason)?

Reply:
torch 1.8.1 is required for compatibility with cuda 11.0.3 
i know we are not having gpu in GA , but having a consistency might be good.

Another thing, where did you find torch to be 1.9.0 , check the imagestream annotation in odh-deployer.
https://github.com/red-hat-data-services/odh-deployer/blob/1d6007f3fea4da6c7935075407ebf6be20f74505/jupyterhub/cuda-11.0.3/manifests.yaml#L129

Reply:
ah I read it from odh-manifests, my mistake

Reply:
ah, I looked in in odh-manifests.

Corrected above with the list from odh-deployer, we still have a discrepancy for numpy and pandas.  We are trying to keep versions consistent across notebooks and consistent with the tooltips, so we need to know why those are different and if they are necessary.

Doesn't look like they were changed in this PR so we need to go back and find where they changed.

Reply:
if you view this pr : https://github.com/red-hat-data-services/odh-deployer/pull/171
it fixes the tooltip as well, i have made sure on the version which we had decided earlier